openapi/components/schemas/GatewayAccountConfig/Worldpay.yaml

```
@@ -67,6 +67,14 @@ allOf:
             type: boolean
             description: Specifies whether to enable stored credentials.
             default: false
+          enforceStoredCredentials:
+            type: string
+            description: Specifies whether to enforce stored credentials.  `mit-enabled` means that stored credentials are sent only on merchant initiated transactions.  This is the expected usage.  `always-enabled` means that stored credentials are also used, if available, for customer facing transactions.
```

Perhaps on a separate lines?

```suggestion
            description: |-
                Specifies whether to enforce stored credentials.

                `mit-enabled` means that stored credentials are sent only on merchant initiated transactions. This is the expected usage.
                `always-enabled` means that stored credentials are also used, if available, for customer facing transactions.
```